wagtail/admin/static_src/wagtailadmin/js/core.js

```diff
@@ -79,6 +79,10 @@ $(function() {
     // Add class to the body from which transitions may be hung so they don't appear to transition as the page loads
     $('body').addClass('ready');
 
+    if (window.location.hash) {
+        $(`a[href='${window.location.hash}']`).tab('show');
```

Comment:
This file isn't part of the build process so it shouldn't be using the ES6 template literal syntax.

Also could you move this next to the `/* tabs */` comment where the other change is made?